src/analysis.rs

```diff
@@ -75,6 +77,7 @@ impl Analysis {
     }
 
     /// Sets the ignorefile basename.
+    #[allow(unused)]
```

Comment:
should this method just be removed? I'm not sure we'd ever want to support alternate ignore file names, which means this can probably just be axed.